src/main/java/io/fusionauth/domain/form/Form.java

```diff
@@ -46,6 +46,19 @@
 
   public FormType type = FormType.registration;
 
+  public Form() {
+  }
+
+  public Form(Form other) {
+    this.data = other.data;
+    this.id = other.id;
+    this.insertInstant = other.insertInstant;
+    this.lastUpdateInstant = other.lastUpdateInstant;
+    this.name = other.name;
+    this.steps = other.steps;
+    this.type = other.type;
+  }
+
```

Comment:
I added a copy constructor so I could create an `updateForm` method in the test builder using the consumer copy pattern we use for other domain objects.